src/Symfony/Component/Stopwatch/StopwatchEvent.php

```diff
@@ -177,7 +177,7 @@ public function getDuration()
 
         for ($i = 0; $i < $left; ++$i) {
             $index = $stopped + $i;
-            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow());
+            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow(), true);
```

Comment:
If I get you right - if whole param would be commented, then the way to retrieve it would be to `func_get_args`.
Big :-1: from my side to that practices.